plugins/txt/plugin/commands/txt_send_cmd.rb

```diff
@@ -0,0 +1,165 @@
+module AresMUSH
+  module Txt
+      class TxtSendCmd
+          include CommandHandler
+# Possible commands... txt name=message; txt =message; txt name[/optional scene #]=<message>
+
+          attr_accessor :names, :message, :scene_id, :scene, :txt, :txt_recipient
+
+        def parse_args
+          if (!cmd.args)
+            self.names = []
+
+          elsif (cmd.args.start_with?("="))
+            self.names = enactor.txt_last
+            self.scene_id = enactor.txt_scene
+            self.message = cmd.args.after("=")
+
+          elsif (cmd.args.include?("="))
+            args = cmd.parse_args(ArgParser.arg1_equals_arg2)
+            # Catch the common mistake of last-paging someone a link.
+            # p http://stuff.com/stuff=this.file
+            if (args.arg1 && (args.arg1.include?("http://") || args.arg1.include?("https://")) )
+              self.names = enactor.txt_last
+              self.message = "#{args.arg1}=#{args.arg2}"
+
+            elsif ( args.arg1.include?("/") )
+              if args.arg1.rest("/").is_integer?
+                self.scene_id = args.arg1.rest("/")
+                self.names = list_arg(args.arg1.first("/"))
+                self.message = trim_arg(args.arg2)
+              elsif args.arg1.rest("/").chr.casecmp?("s")
+                self.scene_id = enactor.room.scene_id
+                self.names = list_arg(args.arg1.first("/"))
+                self.message = trim_arg(args.arg2)
+              end
+            else
+              self.names = list_arg(args.arg1)
+              self.message = trim_arg(args.arg2)
+            end
+
+          else
+            self.names = enactor.txt_last
+            self.scene_id = enactor.txt_scene
+            self.message = cmd.args
+          end
+        end
+
+        def check_txt_target
+            return t('txt.txt_target_missing') if !self.names || self.names.empty?
+            return nil
+        end
+
+
+        def handle
+          # Is scene real and can you text to it?
+          if self.scene_id
+            scene = Scene[self.scene_id]
+            can_txt_scene = Scenes.can_join_scene?(enactor, scene)
```

Comment:
Should check to make sure the scene is not completed already.

Reply:
Isn't line 62 doing that?

Reply:
Yeah, just missed it.  

It looks like one of the text commands has a thing to not log texts, but the others don't.  Didn't know if you meant wanted them all to do that.

Reply:
Yup, all should. Added (I think) everywhere relevant.